<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Obtener Computadoras de un Responsable</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imgs/logo1.png</normaloff>../imgs/logo1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">  background-color: white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="titulo">
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 5px 0 0 0;
</string>
           </property>
           <property name="text">
            <string>Obtener computadoras de un responsable</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
              </property>
              <property name="text">
               <string>Ingrese el nombre del responsable de las computadoras</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_responsable">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
outline:none;
border:1px solid #aaa;
padding:5px;
border-radius: 5px;
color: #333;
margin: 10px 130px 0 130px
}

QLineEdit:focus{
border: 1px solid rgb(225, 81, 81);

}
</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>Ingrese nombre del responsable</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="styleSheet">
            <string notr="true">margin: 0 10px 0 0;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTableWidget" name="tabla_computadoras">
              <property name="styleSheet">
               <string notr="true">QHeaderView::section{
font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
}

QTableWidget QtableCornerButton::section{
font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
border: solid 1px black;


}
QTableWidget{font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
border:solid 1px gray;
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_8">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="respuesta">
           <property name="styleSheet">
            <string notr="true">margin:0 0 5px 0;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_obtener">
           <property name="styleSheet">
            <string notr="true">QPushButton{
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
background-color:rgb(225, 81, 81);
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin:0 150px 10px 150px;}

QPushButton:hover{
  background-color: rgb(113, 40, 40);}</string>
           </property>
           <property name="text">
            <string>Obtener</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
